tslint.json

```diff
@@ -14,6 +14,16 @@
     "ter-arrow-parens": [false],
     "ter-computed-property-spacing": [false],
     "ter-indent": [false],
+    "trailing-comma": [true, {
+        "multiline": {
+          "objects": "always",
+          "arrays": "always",
+          "functions": "always",
+          "typeLiterals": "ignore"
+        },
+        "esSpecCompliant": true
+      }
+    ],
```

Comment:
Fixes error with trailing comma after rest operator.  Without this, TS requires a trailing comma but adding one is also a TS error.